.circleci/config.yml

```diff
@@ -98,7 +98,27 @@ commands:
             ./get_helm.sh --version "${HELM_VERSION}"
             rm ./get_helm.sh
             helm version --short
+  tf_plan_test:
+    steps:
+      - aws-cli/setup:
+         role-arn: "${AWS_IAM_ROLE}"
+         session-duration: "900"
+      - run:
+          name: Terraform plan test
+          working_directory: examples
+          command: |
+            bash tf-plan-test.sh
 jobs:
+  tf-plan-test:
+    docker:
+      - image: cimg/aws:2022.11.1
+    parameters:
+      terraform_version:
+        type: string
+    steps:
+      - checkout
+      - install_tf
```

Comment:
don't you want to pass the terraform_version?

Reply:
I do
https://github.com/dominodatalab/terraform-aws-eks/blob/5896cf2bd812b758d921f004776d564e55df1251/.circleci/config.yml#L213
https://github.com/dominodatalab/terraform-aws-eks/blob/5896cf2bd812b758d921f004776d564e55df1251/.circleci/config.yml#L116

Reply:
you aren't passing it, you are just using defaults everywhere.